motioneye/uploadservices.py

```diff
@@ -381,9 +381,9 @@ def upload_data(self, filename, mime_type, data, ctime, camera_name):
             '',
         ]
 
-        body = '\r\n'.join(body)
+        body = '\r\n'.join(body).encode()
         body += data
-        body += f'\r\n--{self.BOUNDARY}--'
+        body += f'\r\n--{self.BOUNDARY}--'.encode()
```

Comment:
@jmichault 
Shouldn't we do the bytes encoding a single time instead of encoding each appended string individually? Also since `data` is not encoded, isn't this failing anyway, trying to concatenate str and bytes?